components/summary/summary.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Button, Center, Heading, Image, Square } from '@chakra-ui/react';
+import { Flex, Spacer } from '@chakra-ui/react'
+import { useSelector, useDispatch } from 'react-redux';
+import { useRouter } from 'next/router';
+import { NextPage } from 'next';
+import styles from './summary.module.scss';
+import { gameSlicesActions, initialState } from '../../store/game-slice';
+import { RootState } from '../../interfaces/state';
+
+const SummaryLayout: NextPage = () => {
+  const router = useRouter();
+  const dispatch = useDispatch();
+
+  const gameState = useSelector((state: RootState) => state.game);
+    
+    const getCorrectAnswers = () => {
+      const answers =  gameState.playlist.filter((element) => {
+        return element.state === 'correct';
+      })
+      return answers.length;
+    };
+    
+    const getIncorrectAnswers = () => {
+      const answers =  gameState.playlist.filter((element) => {
+        return element.state === 'wrong';
+      })
+      return answers.length;
+    };
```

Comment:
You just need a single method, the other data could be calculated.

Reply:
Ready Sir